listenbrainz/webserver/static/js/src/brainzplayer/SpotifyPlayer.tsx

```diff
@@ -258,7 +258,9 @@ export default class SpotifyPlayer
       if (response.status === 404) {
         // Device not found
         // Wait a second, reconnect and try again
-        await new Promise((resolve) => setTimeout(resolve, 1000));
+        await new Promise((resolve) => {
+          setTimeout(resolve, 1000);
```

Comment:
Apparently returning a value directly from a promise is a no-no, even though in this specific case the code works exactly the same way.
https://eslint.org/docs/latest/rules/no-promise-executor-return